Cython/Utility/ExtensionTypes.c

```diff
@@ -121,7 +121,7 @@ static int __Pyx_PyType_Ready(PyTypeObject *t) {
         (void)__Pyx_PyObject_CallMethod0;
 #endif
 
-    r = PyType_Ready(t);
+        r = PyType_Ready(t);
```

Comment:
This was probably intended (not just indented).
```suggestion
    r = PyType_Ready(t);
```